.travis.yml

```diff
@@ -56,3 +59,18 @@ matrix:
           script: ./docker/deploy.sh $TRAVIS_TAG
           on:
             tags: true
+
+    - name: Coverage
+      rust: nightly
+      script:
+        - curl --location https://github.com/mozilla/grcov/releases/latest/download/grcov-linux-x86_64.tar.bz2 | tar jxf -
+        # These flags are recommended by https://github.com/mozilla/grcov#grcov-with-travis .
+        # vk: I had to remove `-Zpanic_abort_tests -Cpanic=abort` because this would cause compilation
+        # to fail but I haven't investigated more into why.
```

Comment:
Just curious, what was the error message?

Reply:
```
> env CARGO_INCREMENTAL=0 RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests -Cpanic=abort" RUSTDOCFLAGS="-Cpanic=abort" cargo +nightly test
   Compiling driver v0.1.0 (/dex-services/driver)
error: failed to run custom build command for `driver v0.1.0 (/home/work/Gnosis/dex-services/driver)`

Caused by:
  process didn't exit successfully: `/dex-services/target/debug/build/driver-10a168b6e751f4f8/build-script-build` (signal: 6, SIGABRT: process abort signal)
--- stdout
cargo:rerun-if-changed=../dex-contracts/build/contracts/BatchExchange.json
```